carball/tests/analysis_test.py

```diff
@@ -51,6 +53,8 @@ def __test_replays(BASE_DIR):
                                                        sanity_check=SanityChecker())
                 with open(os.path.join(OUTPUT_DIR, 'game.json'), 'w') as f:
                     f.write(MessageToJson(analysis_manager.protobuf_game))
+                pandas = PandasManager.safe_read_pandas_to_memory(BytesIO(analysis_manager.df_bytes))
```

Comment:
I'd rather not use a library's name for a variable (though it obviously does not matter here)